challenge.md

```diff
@@ -5,20 +5,24 @@
 Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
+git clone ssh url
```

Comment:
Just so we're clear, it's not literally `ssh url`, it's whatever that SSH url might be!